files/en-us/web/javascript/reference/statements/async_function/index.html

```diff
@@ -70,13 +70,34 @@ <h2 id="Description">Description</h2>
 }
 </pre>
 
-<p>...is equivalent to:</p>
+<p>...is similar to:</p>
 
 <pre class="brush: js">function <var>foo</var>() {
    <var>return Promise.resolve(1)</var>
 }
 </pre>
 
+<div class="notecard note">
+    <h4 id="checking_equality_with_promise_resolve_vs_async_return">Checking equality with <code>Promise.resolve</code> vs <code>async</code> return</h4>
+    <p>Even though the return value of an async function behaves as if it's wrapped in a <code>Promise.resolve</code>, they are not equivalent.</p>
+    <p>An async function will return a different <i>reference</i>, whereas <code>Promise.resolve</code> returns the same reference if the given value is a promise.</p>
```

Comment:
```suggestion
    <p>An async function will return a different <em>reference</em>, whereas <code>Promise.resolve</code> returns the same reference if the given value is a promise.</p>
```

(Personally I like `<i></i>` better than `<em></em>`, but MDN style is to use `<em></em>`)